crates/hc_bundle/src/packing/wasmer_wamr.rs

```diff
@@ -0,0 +1,12 @@
+#![cfg(feature = "wasmer_wamr")]
+
+use crate::error::HcBundleError;
+use mr_bundle::{Bundle, Manifest};
+use std::path::Path;
+
+pub(super) async fn build_preserialized_wasm<M: Manifest>(
+    _target_path: &Path,
+    _bundle: &Bundle<M>,
+) -> Result<(), HcBundleError> {
+    unimplemented!("The feature flag 'wasmer_sys' must be enabled to support compiling wasm");
```

Comment:
Oh right, this is the thing that isn't supported. So maybe my questions above about supporting both don't make sense. So why do we need a dependency on wamr at all here?

Reply:
Ha oh yeah whoops. I'll remove the flag entirely there's no need for it in hc_bundle.

Reply:
Actually remembering my reasoning -- by excluding the flags in hc & hc_bundle, we're not able to run a cargo build for the entire workspace with the `wasmer_wamr` flag. Not sure if that's desired.

Reply:
As in, we end up with two wasmers getting built? Even though they end up in different binaries... Hmm, that sucks! Yeah Cargo can make things quite difficult sometimes.

I wonder if a wasmer dependency doesn't really belong in this tool then. I get that it's convenient though.

Reply:
If we want to keep the functionality to precompile & serialize a wasm to a file and then write the path in the dna manifest, then I think it makes sense to keep it in hc_bundle. The actual logic for compiling the wasm is contained in holochain_wasmer_host, which also seems reasonable. 

Currently the functionality doesn't work fully so I assume no one is using it. It may still be useful as an alternative to the interpreter, but maybe that replaces its utility. 

Though if we were to remove that functionality, I should probably remove it fully as a standalone PR. Otherwise I'd like to keep it and fix it when I have time.